### ISSUE TITLE
TwoSideAttack: Behavior tree does not update when ball goes out and comes back in

### ISSUE THREAD

**Issue by joshhting** [RoboJackets/robocup-software#1121]
I ran TwoSideAttack, robot 5 passed the ball to robot 4 but missed because it didn't finish turning yet, so the ball went out of bounds. I moved the ball back into the field, but they won't transition out of CoordinatedPass::failed
![screenshot from 2018-04-18 21-52-37](https://user-images.githubusercontent.com/8889217/38967007-11e0b3b6-4353-11e8-98ec-739d3bfe9e79.png)

**Comment by jgkamat** [comment]
Yup, we have a [transition for completed, but not for failed](https://github.com/RoboJackets/robocup-software/blob/master/soccer/gameplay/plays/offense/two_side_attack.py#L50) which needs to be added :)
